io.github.yesser_studios.Pong.yml

```diff
@@ -0,0 +1,51 @@
+id: io.github.yesser_studios.Pong
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+finish-args:
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=host-os:ro
+
+command: run.sh
+modules:
+  - name: dotnet
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/dotnet8/bin/install.sh
+
+  - name: pong
+    buildsystem: simple
+    build-options:
+      prepend-path: "/usr/lib/sdk/dotnet8/bin"
+      append-ld-library-path: "/usr/lib/sdk/dotnet8/lib"
+      prepend-pkg-config-path: "/usr/lib/sdk/dotnet8/lib/pkgconfig"
+      arch:
+        aarch64:
+          env:
+            RUNTIME: linux-arm64
+        x86_64:
+          env:
+            RUNTIME: linux-x64
+    build-commands:
+      - cp -r --remove-destination ./bin/* ${FLATPAK_DEST}/bin/
+      - install -Dm644 flatpak/io.github.yesser_studios.Pong.desktop -t ${FLATPAK_DEST}/share/applications/
+      - install -Dm644 flatpak/io.github.yesser_studios.Pong.png -t ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps
+      - install -Dm644 flatpak/io.github.yesser_studios.Pong.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo
+      - echo "dotnet /app/bin/Pong.Desktop.dll" > run.sh
+      - chmod +x ./run.sh
+      - cp -r --remove-destination ./run.sh ${FLATPAK_DEST}/bin/
+    sources:
+      - type: git
+        url: https://github.com/yesser-studios/Pong.git
+        tag: v1.2.0
+      - type: archive
+        url: https://github.com/yesser-studios/Pong/releases/download/v1.2.0/pong-linux.tar.xz
```

Comment:
so what happen on aarch64?

Reply:
I'm not sure, I'll try testing on my Arch VM when I get home.

Reply:
The only difference should be the missing deps for MonoGame that don't get included in the build. If that is an issue (the app won't run with the x64 ones), I'll have to restrict the package to x64.

Reply:
I've tested and can confirm this flatpak can be installed and run on an aarch64 Debian VM.